vc-verifier/kotlin/README.md

```
@@ -117,6 +125,68 @@ refer to the implementation classes for complete rules.
     * Verifies signature using resolved key from `cnf`
 * **Validation class:** `SdJwtValidator`
 
+### 3.4 CWT VC Validation
+
+CWT VC validation handles CBOR-encoded Verifiable Credentials encapsulated in a `COSE_Sign1` structure,
+as defined by the IETF CWT and COSE specifications.
+
+* **Input validation:**
+
+    * Rejects empty credentials
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)